src/protocol/sort/compose.rs

```diff
@@ -4,24 +4,27 @@ use embed_doc_image::embed_doc_image;
 
 use super::{apply::apply, shuffle::unshuffle_shares, ComposeStep::UnshuffleRho};
 
+#[embed_doc_image("compose", "images/sort/compose.png")]
 /// This is an implementation of Compose (Algorithm 5) found in the paper:
 /// "An Efficient Secure Three-Party Sorting Protocol with an Honest Majority"
 /// by K. Chida, K. Hamada, D. Ikarashi, R. Kikuchi, N. Kiribuchi, and B. Pinkas
 /// <https://eprint.iacr.org/2019/695.pdf>
+///
 /// This protocol composes two permutations by applying one secret-shared permutation(sigma) to another secret-shared permutation(rho)
 /// Input: First permutation(sigma) i.e. permutation that sorts all i-1th bits and other permutation(rho) i.e. sort permutation for ith bit
 /// Output: All helpers receive secret shares of permutation which sort inputs until ith bits.
-
-#[embed_doc_image("compose", "images/sort/compose.png")]
+///
 /// This algorithm composes two permutations (`rho` and `sigma`). Both permutations are secret-shared,
 /// and none of the helpers should learn it through this protocol.
 /// Steps
-/// ![Compose steps][compose]
+///
 /// 1. Generate random permutations using prss
 /// 2. First permutation (sigma) is shuffled with random permutations
 /// 3. Reveal the permutation
 /// 4. Revealed permutation is applied locally on another permutation shares (rho)
 /// 5. Unshuffle the permutation with the same random permutations used in step 2, to undo the effect of the shuffling
+///
+/// ![Compose steps][compose]
```

Comment:
What does this do?

Reply:
This shows the image in the UI like this
![Uploading Screenshot 2022-12-07 at 12.13.35 PM.png…]()